src/navigation.tpl

```diff
@@ -195,14 +189,16 @@
 <!-- SECTION -->
     <li class="section"><h4 class="header toggle-target">Reference</h4>
         <ul>
+        <li><a href="reference-install.html">Install Juju</a></li>
+        <li><a href="introducing-2.html">Introducing Juju 2.0!</a></li>
+        <li><a href="whats-new.html">What's new in 2.1</a></li>
```

Comment:
I'm +1 when we do the what's new with 2.2 at the top? I guess I feel like there's a time window here and 2.1 is old enough that it pushes more important stuff down.